package.json

```diff
@@ -82,7 +82,8 @@
     "**/@storybook/**/ansi-regex": "^5.0.1",
     "**/jsprim/json-schema": "^0.4.0",
     "**/@storybook/**/yuidocjs/markdown-it": "^12.3.2",
-    "**/ember-cli/markdown-it-terminal/markdown-it": "^12.3.2"
+    "**/ember-cli/markdown-it-terminal/markdown-it": "^12.3.2",
+    "**/ember-cli-mirage/@embroider/macros": "^1.0.0"
```

Comment:
This resolution is needed because there's a current issue when building with multiple versions of `@embroider/macros`. See this [issue](https://github.com/embroider-build/embroider/issues/592) and this [one](https://github.com/miragejs/ember-cli-mirage/issues/2343) for more info.